src/allmydata/interfaces.py

```diff
@@ -2936,3 +2936,68 @@ class IConnectionStatus(Interface):
         (pending, connected, refused, or other errors).
         """)
 
+
+class ITahoeUsagePolicy(Interface):
```

Comment:
I have started calling these kinds of things "storage authorizers" in design docs.  I'm not super happy with that name or this one.  Maybe we can brainstorm on this a bit.

Reply:
Sure. The above is just what I came up with that sounded better than "EconomicPlugin" which doesn't say much either ;)